librz/core/cmd_descs.yaml

```
@@ -590,9 +590,48 @@
         - name: project.rzdb
           type: RZ_CMD_ARG_TYPE_FILE
 - name: q
-  cname: cmd_quit
-  summary: Quit program with a return value
-  type: RZ_CMD_DESC_TYPE_OLDINPUT
+  summary: Quit rizin
+  subcommands:
+    - name: "q"
+      cname : cmd_quit
+      summary: quit rizin
+      args: []
+    - name: "q!"
+      cname: cmd_force_quit
+      summary: Force quit rizin
+      args: []
+    - name: "q!!"
+      cname: cmd_force_quit_without_history
+      summary: force quit rizin without saving history
+      args: []
+    - name: "q"
+      cname: quit_with_preference
+      summary: Quit rizin and choose to save projects and kill the process
```

```suggestion
      summary: Quit rizin and choose to kill the process and save projects
```
Because the first <yn> refers to the kill, the second to the projects.

---

Oh, yeah, missed that one. All of the summaries seem to be fine.